scripts/jenkins/msvc.groovy

```diff
@@ -26,6 +26,7 @@ node('win && conan') {
 
         stage 'Test (Win)'
         build.win 'build', 'tests'
+        build.win 'build', 'ctest'
```

Comment:
This is release mode I hope; it could take too much time in debug even for non-LARGE ctests.